website_product_availability/models/product.py

```diff
@@ -79,3 +79,84 @@ def _get_availability(self, cr, uid, ids, name, args, context=None):
     _default = {
         'low_stock': 0,
     }
+
+
+class LocationQuants(osv.Model):
+    _name = 'location.quants'
+
+    _columns = {
+        'product_id': fields.many2one('product.template'),
```

Comment:
You need to add a string attribute to this fields
``` python
string="Product"
```
because headers on tree appear as 'Unknown'
![image](https://www.evernote.com/l/AFyRwXBrt5BMC4bVlQtD2jMfsbhzN-Nbp7oB/image.png)